app/styles/pages/_media-browse.scss

```diff
@@ -1,3 +1,8 @@
+.noUi-handle,
```

Comment:
Unsure of whether to keep it in here or if it deserves its own component, since sliders may get used outside of browse pages.